saas/api/billing.py

```diff
@@ -254,15 +254,15 @@ def post(self, request, *args, **kwargs):
 
         for row in filed:
             try:
+                # TODO should I change the interface here too?
```

Comment:
This is a case where if the first line of the CSV contains column names, it might be useful.